webview/src/components/file-input.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import { Add } from "svelte-codicons"
+  export let id: string
+  export let required: boolean = false
+  type FileEventTarget = EventTarget & { files: FileList }
+
+  const defaultText = "No file chosen"
+
+  function handleOnChange(e: Event) {
+    const target = e.target as FileEventTarget
+    const fileChosen = document.getElementById(`${id}-file-chosen`)
+
+    if (fileChosen instanceof HTMLElement) {
+      const file = target.files[0]
+      fileChosen.textContent = file?.name ?? defaultText
+    }
+  }
+</script>
+
+<div class="file-input">
+  <input {id} on:change={handleOnChange} type="file" {required} name={id} hidden />
```

Comment:
Why do we need to add IDs everywhere?

Reply:
See https://github.com/temporalio/vscode-debugger-plugin/pull/18#discussion_r984974756.